src/NexusMods.App.UI/WorkspaceSystem/PanelTabHeader/PanelTabHeaderView.axaml

```diff
@@ -7,35 +7,41 @@
     xmlns:reactive="http://reactiveui.net"
     xmlns:workspace="clr-namespace:NexusMods.App.UI.WorkspaceSystem"
     xmlns:icons="clr-namespace:NexusMods.Icons;assembly=NexusMods.Icons"
-    mc:Ignorable="d" d:DesignWidth="232" d:DesignHeight="52"
+    xmlns:controls="clr-namespace:NexusMods.App.UI.Controls"
+    mc:Ignorable="d" d:DesignWidth="400" d:DesignHeight="60"
     x:Class="NexusMods.App.UI.WorkspaceSystem.PanelTabHeaderView">
 
     <Design.DataContext>
         <workspace:PanelTabHeaderDesignViewModel />
     </Design.DataContext>
 
-    <Border x:Name="Container">
-        <StackPanel Orientation="Horizontal">
-            <StackPanel Orientation="Horizontal"
-                        Classes="Spacing-2"
-                        Height="52"
-                        Margin="24, 0, 24, 0">
-                <StackPanel Orientation="Horizontal"
-                            Classes="Spacing-1_5"
-                            VerticalAlignment="Center">
-                    <icons:UnifiedIcon x:Name="Icon"/>
-                    <TextBlock x:Name="TitleTextBlock"
-                               Classes="BodyMDBold"
-                               MaxWidth="122"
-                               TextWrapping="NoWrap"
-                               TextTrimming="CharacterEllipsis"/>
-                </StackPanel>
-
-                <Button x:Name="CloseTabButton" Classes="PanelTitlebar Close" />
-
-            </StackPanel>
-
-        </StackPanel>
+    <Border x:Name="Container" HorizontalAlignment="Left">
+
+        <DockPanel MinWidth="100" MaxWidth="260"
+                   Height="36"
+                   LastChildFill="False">
+
+            <controls:StandardButton x:Name="CloseTabButton"
+                                     ShowIcon="IconOnly"
+                                     LeftIcon="{x:Static icons:IconValues.Close}"
+                                     Fill="None"
+                                     Size="Small"
+                                     DockPanel.Dock="Right" Margin="0, 0, 8, 0" />
+
+            <DockPanel
+                VerticalAlignment="Center"
+                HorizontalAlignment="Left"
+                Margin="16,0,0,0">
+                <icons:UnifiedIcon x:Name="Icon" DockPanel.Dock="Left" />
+                <TextBlock x:Name="TitleTextBlock"
+                           Theme="{StaticResource BodySMBoldTheme}"
+                           TextWrapping="NoWrap"
+                           TextTrimming="CharacterEllipsis"
+                           Margin="8,0,8,0" />
+            </DockPanel>
+
+        </DockPanel>
+
```

Comment:
Why did you go with DockPanel here?

Reply:
I couldn't quite get the positions and the widths to calculate correctly otherwise. A few things going on with being able to set a min and max (which will change over time) and the Close button to stay on the right, and the TextBlock to trim correctly depending on the width